astropy/coordinates/distances.py

```diff
@@ -210,7 +210,7 @@ def parallax(self):
         """The parallax angle as an `~astropy.coordinates.Angle` object"""
         return Angle(self.to(u.milliarcsecond, u.parallax()))
 
-
+# Looks like this function isn't used anywhere. Should it be removed?
```

Comment:
Correct, looks like I didn't notice it in https://github.com/astropy/astropy/commit/a155850ea9c0ffc41b26c26ff11fbdc6f570d6c7, where I made `Distance` a subclass of `SpecificTypeQuantity`, which made this unnecessary.

Do delete it as part of this PR!

Reply:
Done, function has been removed.